### PR TITLE
Review of exercises for Visualisation course 2024

### PR DESCRIPTION
**Pull request by AnnikaLau**

### CODE REVIEW

**Review by mathause — APPROVED** [review]
Was it too fast? :rofl: You could probably do `%timeit ...` instead.

**Comment by AnnikaLau** [comment]
> Was it too fast? 🤣 You could probably do `%timeit ...` instead.

Haha no but the description said "Create a python list with a million integers" and not 100,000. Thought it should be consistent ;-)

File: environment.yml
Comment (mathause) [inline environment.yml:1]: Could you add comments why you are pinning the packages, so it's easier to un-pin them again?

Comment (AnnikaLau) [reply environment.yml:1]: You are right. I added comments in 277603c5609f6b4ccfdeb81e8a51f0f01c94d0fe.
I will have a look into this issue after the course next week.

Comment (mathause) [reply environment.yml:1]: thanks

File: Part3_icon-vis/ex3_0_psyplot.ipynb
Comment (mathause) [inline Part3_icon-vis/ex3_0_psyplot.ipynb:1]:
```diff
-To close figure "ds"
+To close the figure "ds"
```
?

**Review by mathause — COMMENTED** [review]
Looks good (but I went over it fast). Thanks!

**Comment by AnnikaLau** [comment]
Thank you @mathause. I made all your requested changes. Would you be okay with merging?

**Review by mathause — APPROVED** [review]
Sure!

**Pull request merged**